backend/src/ee/services/ldap-config/ldap-config-service.ts

```diff
@@ -0,0 +1,449 @@
+    return getLdapCfg({
+      orgId
+    });
+  };
+
+  // eslint-disable-next-line
+  const getLdapPassportOpts = (req: FastifyRequest, done: any) => {
+    const { organizationSlug } = req.body as {
+      organizationSlug: string;
+    };
+
+    const boot = async () => {
+      try {
+        const organization = await orgDAL.findOne({ slug: organizationSlug });
+        if (!organization) throw new BadRequestError({ message: "Org not found" });
+
+        const ldapConfig = await getLdapCfg({
+          orgId: organization.id,
+          isActive: true
+        });
+        req.ldapConfig = ldapConfig;
+
+        const opts = {
+          server: {
+            url: ldapConfig.url,
+            bindDN: ldapConfig.bindDN,
+            bindCredentials: ldapConfig.bindPass,
+            searchBase: ldapConfig.searchBase,
+            searchFilter: "(uid={{username}})",
+            searchAttributes: ["uid", "uidNumber", "givenName", "sn", "mail"],
+            ...(ldapConfig.caCert !== ""
+              ? {
+                  tlsOptions: {
+                    ca: [ldapConfig.caCert]
+                  }
+                }
+              : {})
+          },
+          passReqToCallback: true
+        };
+
+        // eslint-disable-next-line
+          done(null, opts);
+      } catch (err) {
+        logger.error(err);
+        // eslint-disable-next-line
+          done(err);
+      }
+    };
+
+    process.nextTick(async () => {
+      await boot();
+    });
+  };
+
+  const ldapLogin = async ({ externalId, username, firstName, lastName, emails, orgId, relayState }: TLdapLoginDTO) => {
+    const appCfg = getConfig();
+    let userAlias = await userAliasDAL.findOne({
+      externalId,
+      orgId,
+      aliasType: AuthMethod.LDAP
+    });
+
+    const organization = await orgDAL.findOrgById(orgId);
+    if (!organization) throw new BadRequestError({ message: "Org not found" });
+
+    if (userAlias) {
+      await userDAL.transaction(async (tx) => {
+        const [orgMembership] = await orgDAL.findMembership({ userId: userAlias.userId }, { tx });
+        if (!orgMembership) {
+          await orgDAL.createMembership(
+            {
+              userId: userAlias.userId,
+              orgId,
+              role: OrgMembershipRole.Member,
+              status: OrgMembershipStatus.Accepted
+            },
+            tx
+          );
+        } else if (orgMembership.status === OrgMembershipStatus.Invited) {
+          await orgDAL.updateMembershipById(
+            orgMembership.id,
+            {
+              status: OrgMembershipStatus.Accepted
+            },
+            tx
+          );
+        }
+      });
+    } else {
+      userAlias = await userDAL.transaction(async (tx) => {
+        const uniqueUsername = await normalizeUsername(username, userDAL);
+        const newUser = await userDAL.create(
+          {
+            username: uniqueUsername,
+            email: emails[0],
```

Comment:
`isAccepted: true` is missing

Reply:
At this point actually there shouldn't be `isAccepted` (i.e. it is false) because the user will later complete the signup flow and only at that point this becomes `true`.